plenary/plenary-38.yaml

```diff
@@ -0,0 +1,554 @@
+        date_effective: 2019-10-17
+        message: Considering Resolution 2018-19 Re-establishment of ISO/TC 154/WG 4 with the title of “e-document”
+
+      - type: considering
+        date_effective: 2019-10-17
+        message: Considering that WG4 will be established when ISO 36001 is approved
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity.
+
+    actions:
+      - type: resolves
+        date_effective: 2019-10-17
+        message: resolves to appoint Mr. Ronald Henry Tse from Calconnect as the co-convener of ISO/TC 154/WG4
+
+      - type: resolves
+        date_effective: 2019-10-17
+        message: resolves to appoint Mr. Jamie Clark from OASIS as the co-convener of ISO/TC 154/WG4. Their terms are for 3 years until the end of 2022.
+
+
+  - category: Resolutions related to liaison membership
+    dates:
+      - 2019-10-17
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Re-submission of ISO/NP PAS 15000-1 and ISO/NP PAS 15000-2"
+    identifier: 2019-17
+    considerations:
+      - type: considering
+        date_effective: 2019-10-17
+        message: acknowledges that the ballot of ISO/NP PAS 15000-1 and ISO/NP PAS 15000-2 was conducted under an inappropriate procedure
+
+      - type: considering
+        date_effective: 2019-10-17
+        message: agrees that OASIS, as a Category A liaison, will instead submit both drafts directly to ISO/CS as DIS via the fast-track procedure outlined in ISO/IEC Directives, Part 1, F.2.1.1.
+
+      - type: considering
+        date_effective: 2019-10-17
+        message: assigns the project to WG7
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity.
+
+
+  - category: Resolutions related to liaison membership
+    dates:
+      - 2019-10-18
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Resolves to initiate a ballot to call for liaison representatives from ISO/TC154 to the ISO internal committee liaisons"
+    identifier: 2019-18
+    considerations:
+      - type: considering
+        date_effective: 2019-10-17
+        message: considering that A-liaison members of ISO/TC 154 have nominated liaison representatives to ISO/TC 154
+
+      - type: considering
+        date_effective: 2019-10-17
+        message: Considering the increase of ISO internal committee liaisons with ISO/TC 154
+
+      - type: considering
+        date_effective: 2019-10-17
+        message: Considering the liaison representatives have been nominated to ISO/PC 317, ISO/TC 215, but no liaison representatives have been nominated to other ISO internal liaisons
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity.
+
+      - type: resolves
```

Comment:
@anermina the `type: resolves` should be under `actions:` instead of `approvals:`. The few actions below this text seem to have this issue.